README.md

```diff
@@ -0,0 +1,13 @@
+# CytoCherryPick
```

Comment:
Consider adding a description for where the project name comes from (for example, a phrase definition, or cultural reference).

Reply:
Updated!